internal/converter/internal/promtailconvert/testdata/azure.alloy

```diff
@@ -17,7 +17,9 @@ local.file_match "fun" {
 }
 
 loki.source.file "fun" {
-	targets               = local.file_match.fun.targets
-	forward_to            = []
+	targets    = local.file_match.fun.targets
+	forward_to = []
+
+	file_match { }
```

Comment:
Hmm, something is not right with the defaults I think. Ideally there should be no change needed for the converters.